src/backend/stream.rs

```diff
@@ -112,8 +119,22 @@ impl Drop for Device {
     }
 }
 
+enum RingBufferConsumer {
+    IntegerRingBufferConsumer(ringbuf::Consumer<i16>),
+    FloatRingBufferConsumer(ringbuf::Consumer<f32>)
+}
```

Comment:
`RingBufferConsumer::IntegerRingBufferConsumer` or `RingBufferConsumer::FloatRingBufferConsumer`is a bit long.

Reply:
You could `use RingBufferConsumer::*` to avoid repeating the enum name everywhere.

Reply:
Yep, thanks.